android/assets/jsons/Civ V - Vanilla/Ruins.json

```diff
@@ -0,0 +1,64 @@
+[
+    {
+        "name": "freeCulture",
+        "notification": "We have discovered cultural artifacts in the ruins! (+20 culture)",
+        "uniques": ["Gain [20] [Culture]"]
+    },
+    {
+        "name": "joinWorker",
+        "notification": "A worker has joined us!",
+        "uniques": ["Free [Worker] found in the ruins"],
+        "excludedDifficulties": ["Prince", "King", "Emperor", "Immortal", "Deity"]
+    },
+    {
+        "name": "joinSettler",
+        "notification": "A settler has joined us!",
+        "uniques": ["Free [Settler] found in the ruins"],
+        "excludedDifficulties": ["Warlord","Prince","King","Emperor","Immortal","Deity"]
+    },
+    {
+        "name": "freeXP",
+        "notification": "An ancient tribe trained us in their ways of combat!",
+        "uniques": ["This Unit gains [10] XP"]
+    },
+    {
+        "name": "freePop",
+        "notification": "We have found survivors in the ruins! Population added to [cityName].",
+        "uniques": ["[+1] population in a random city"] // This can't be easily added to cityFilter, as it is non-deterministic
+    },
+    {
+        "name": "freeGold",
+        "notification": "We have found a stash of [goldAmount] Gold in the ruins!",
+        "uniques": ["Gain [50]-[100] [Gold]"]
+    },
+    {
+        "name": "freeAncientTech",
+        "notification": "We have discovered the lost technology of [techName] in the ruins!",
+        "uniques": ["[1] free random researchable Techs from the [Ancient era]"]
+    },
+    {
+        "name": "unitUpgrade",
+        "notification": "Our unit finds advanced weaponry hidden in the ruins!",
+        "uniques": ["This Unit finds advanced weapons"]
```

Comment:
I think this should be phrased as "this unit upgrades", the notification can be flavor text but I think the unique should be mechanical effect

Reply:
Problem is that there is already an unique for "this unit upgrades", which follows the normal upgrade path. I'll probably change it to "this unit upgrades including special upgrades" or something to still clearly differentiate between the two